core/src/test/java/org/owasp/dependencycheck/analyzer/DependencyCheckPropertiesTest.java

```diff
@@ -89,7 +89,7 @@ private boolean isATestAnalyzer(Class<?> clazz) {
     }
 
     public Set<Class<?>> findAllClasses(String packageName) throws IOException {
-        String parsedPackageName = packageName.replaceAll("[.]", File.separator);
+        String parsedPackageName = packageName.replaceAll("[.]", File.separator.replaceAll("\\\\","\\\\\\\\"));
```

Comment:
```suggestion
        String parsedPackageName = packageName.replaceAll("[.]", "/");
```

As the resourceloader is used the proper fix should be replacing `.` by forward slash see [getResources javadoc](https://docs.oracle.com/javase/8/docs/api/java/lang/ClassLoader.html#getResources-java.lang.String-)